extensions/zerion/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
 # Zerion Changelog
 
-## [Initial Version] - 2024-01-18
\ No newline at end of file
+## [Initial Version] - 2024-01-18 (1.0.0)
+
+## [Zerion Client ] - 2024-12-17 (2.0.0)
+- View wallet portfolio
+- Search tokens, addresses or domains
+- Show Wallet total value in the menu bar
+- Safe multiple addresses for quick access
```

Comment:
```suggestion
## [Zerion Client] - {PR_MERGE_DATE}
- View wallet portfolio
- Search tokens, addresses or domains
- Show Wallet total value in the menu bar
- Safe multiple addresses for quick access

## [Initial Version] - 2024-01-18
```

Incorrect order. The new changelog should be on the top. The version is nonsense and unneeded.